packages/tabsheet/theme/lumo/vaadin-tabsheet.js

```diff
@@ -0,0 +1,5 @@
+import '@vaadin/tabs/theme/lumo/vaadin-tabs.js';
+import '@vaadin/tabs/theme/lumo/vaadin-tab.js';
```

Comment:
nit: this import isn't really needed, as the one for `vaadin-tabs` includes it.

Reply:
Done